source/includes/_openbanking.md

```diff
@@ -0,0 +1,374 @@
+#Open Banking API
+
+> Base Open Banking URL in Sandbox
+
+```shell
+https://openbanking.sandbox.transferwise.tech
+```
+
+> Base Open Banking URL in Production
+
+```shell
+https://openbanking.transferwise.com
+```
+
+The TransferWise Open Banking API is a collection of RESTful APIs that enable Third Party Providers (TPPs) to access account information, initiate payments and confirm availability of funds on behalf of TransferWise customers. The API implementation follows the [Open Banking UK standard](https://standards.openbanking.org.uk/api-specifications/), and TransferWise is a registered member of the [Open Banking Directory](https://www.openbanking.org.uk/providers/directory/).
+
+Through this API you can either query account information, if you're an Account Information Service Provider (**AISP**), initiate payments, if you're a Payment Initiation Service Provider (**PISP**), or confirm the availability of funds, if you're a Card Based Payment Instrument Issuer (**CBPII**). We are currently supporting version [3.1.1](https://standards.openbanking.org.uk/api-specifications/read-write-specs/v3-1-1/) of the Open Banking API.
+
+## Prerequisites
+
+> Well-Known Open Banking URL
+
+```shell
+https://sandbox.transferwise.tech/openbanking/.well-known/openid-configuration
+```
+
+There's a couple of things you should be aware of in order to connect to the TransferWise Open Banking API:
+
+* Being a registered TPP under the Open Banking Directory will ease the integration process.
+* We are using [mutualTLS](https://en.wikipedia.org/wiki/Mutual_authentication) as the means for authentication. Moreover, the `CN` of your client certificate is expected to match the `clientId` under which you are registered. 
+* You need to register your client, before you'll be able to call any of the APIs. Check out the [Sandbox Access](#open-banking-api-sandbox-access) first.
+
+For a detailed description of requirements and supported algorithms please check out the Well-Known Open Banking URL.
+
+## Sandbox Access
+
+We highly recommend that you get started with connecting to our sandbox first, before moving on to production. To get started with the registration process drop us an email to `openbanking@transferwise.com`
```

Comment:
to [openbanking@transferwise.com](mailto:openbanking@transferwise.com) ?

Reply:
Yup. The mailing list already exists. Makes sense to keep contacts separate from other public API related queries.

Reply:
Ok, github markdown eat my point. My point was use link annotation and mailto: form for improved usability.